lib/assets/javascripts/views/devise/registrations/edit.js

```diff
@@ -17,4 +19,29 @@ $(() => {
     $('#org-controls .combobox-clear-button').hide();
     $('#other_org_toggle a').hide();
   }
+
+  // If the user has changed their email address display the password
+  // confirmation modal on form submission
+  $('#personal_details_registration_form [type="submit"]').click((e) => {
+    const newEmail = $('#personal_details_registration_form #user_email');
```

Comment:
although the selector works, is not needed to select the id user_mail given the parent personal_details_registration_form since IDs must be unique across the whole page.